tests/python/unittest/test_runtime_rpc.py

```diff
@@ -448,10 +448,10 @@ def check_remote():
 
 
 @tvm.testing.requires_rpc
-def test_rpc_tracker_register():
+@pytest.mark.parametrize("device_key", ["test_device", "127.0.0.1:5555"])
```

Comment:
Is it a complete list with all possible keys which `adb devices` can return? Not clear from this added key, why previous version with split doesn't work.

Reply:
I'm not sure. I want to fix exactly the current case. 
The issue in that case is in format of key `hexagon-dev.192.168.0.143:5555:0.513619`.

Before the fix this test was falling with "127.0.0.1:5555", after it works

Reply:
Hm... It looks strange that the test has failed before with "127.0.0.1:5555". I wrote the following code:
```python
key = "127.0.0.1:5555"

print(key.split(":")[0])
print(key.rsplit(":", 1)[0])
```

And expectedly received the following result:
```
127.0.0.1
127.0.0.1
```

What is the problem to add one more key, e.g. `hexagon-dev.192.168.0.143:5555:0.513619`?

Reply:
> What is the problem to add one more key, e.g. hexagon-dev.192.168.0.143:5555:0.513619?
It is not device key, it is internal representation inside implementation.

Reply:
Thank you for clarification! In verbal discussion, decided to fix all usage of key split in TVM code, create a function which have to do this split and write simple unit tests on it. To show that the key might has three parts separated by `:`: `host:port:unique_random_key`.